pages/participantinfo.tsx

```diff
@@ -28,10 +28,21 @@ import { useState } from "react";
 import { GetServerSideProps } from "next"; // Get server side props
```

Comment:
@mahadzaryab1 mentioned that for typescript files, they should be camelCase so could you change it to `participantInfo.tsx`?

Reply:
@jason17h Bump! not sure if you saw this

Reply:
Does the same thing apply to files in the `api` directory since these are also used as the API route? I tried changing it to `participantInfo` but the URL automatically changes back to `participantinfo` and it gives me this error page:
<img width="561" alt="Screen Shot 2021-08-10 at 11 14 08 AM" src="https://user-images.githubusercontent.com/42329517/128892870-a8af7634-2559-4c97-b019-325d7b9d8052.png">
Maybe using a dash would be better for readability? i.e., `participant-info.tsx`